### ISSUE TITLE
💬 TALK - Only GCA related

### ISSUE THREAD

**Issue by FrutyX** [DinoDevs/GladiatusCrazyAddon#308]
A talk issue, suitable for GCA development.

Sometimes, you just have a simple question, or you want to suggest something, but it isn't completely up to create a whole new Issue, this is the right place to talk about it.

## **Rules:**

**1.** As the issue name suggest, only GCA related talk here (applies to links too), Gladiatus topic is still tolerable, nothing else other than that.

**2.** No vulgarity or spam, keep this Issue clean and when you forget to mention something, just edit your previous post.

**3.** If you found a bug or want to make a suggestion for a feature or enhancement, create a **New issue**.

**4.** If you are having problems with GCA, make sure to read our <a href="https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/Troubleshooting.md">Troubleshooting guide</a> first.

**5.** You are free to use emojis and upload images, all in a reasonable amount.

**6.** Be kind and helpful.
##

In case of need, more rules might get added. Comments that will not respect these rules will be deleted.

For developer questions, you can also check <a href="https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/developers/for-developers.md">For developers</a>.

## Archive

<details>
<summary> Archived talk Issues </summary>
<p></p>

- #161
- #156 
- #174 
- #175 
- #317 
...

</details>

**Comment by FrutyX** [comment]
@GramThanos , @GreatApo 

There is no point in making a feature request, because I know (98%) that this will never be implement, the reason is simple - too much work, you have your own real life and you are not getting paid enough (heh).

I found the fan made avatars in files, well, a long time ago.

At first, I thought to myself that it's a shame they are not used (as far as I know), so a small fun thing would be, to add them via the addon and you can change your in-game avatar for these, of course you'll be the only one to see that, the opponent will see only Gladiatus default costume. This would be the easiest solution to use them.

Then I got a better idea, but the amount of work for this is insane. Back in the day, where GCA development was most active, this would be maybe possible, but now? I don't know, the motivation and time... I don't blame you.

So the second idea was all about Global Arena, what if we added these custom avatars (costumes) as tier rewards? For completing achievements, like "Win 50 fights to unlock this costume" or "Spend XX/XXX time in top 3 to unlock this costume".

There is still the problem, that you'll be the only one to see them, but, I don't know the capabilities of Global Arena, it could be possible to see the costumes, but only in Global Arena, the implementation is kinda difficult. 

A database (MySQL?) would be needed to save the unlocked costumes for each player and then show them in the Global Arena, hard, not impossible (guessing).

In the end, even if they were unlockable as tier rewards, only for you to be visible, it would be still nice, even without the database thing, it's still a lot of work.

I wanted to share with you this, even when I know, that this is not possible.

**Comment by GreatApo** [comment]
> @GramThanos , @GreatApo 
> 
> There is no point in making a feature request, because I know (98%) that this will never be implement, the reason is simple - too much work, you have your own real life and you are not getting paid enough (heh).
> 
> I found the fan made avatars in files, well, a long time ago.
> 
> At first, I thought to myself that it's a shame they are not used (as far as I know), so a small fun thing would be, to add them via the addon and you can change your in-game avatar for these, of course you'll be the only one to see that, the opponent will see only Gladiatus default costume. This would be the easiest solution to use them.
> 
> Then I got a better idea, but the amount of work for this is insane. Back in the day, where GCA development was most active, this would be maybe possible, but now? I don't know, the motivation and time... I don't blame you.
> 
> So the second idea was all about Global Arena, what if we added these custom avatars (costumes) as tier rewards? For completing achievements, like "Win 50 fights to unlock this costume" or "Spend XX/XXX time in top 3 to unlock this costume".
> 
> There is still the problem, that you'll be the only one to see them, but, I don't know the capabilities of Global Arena, it could be possible to see the costumes, but only in Global Arena, the implementation is kinda difficult. 
> 
> A database (MySQL?) would be needed to save the unlocked costumes for each player and then show them in the Global Arena, hard, not impossible (guessing).
> 
> In the end, even if they were unlockable as tier rewards, only for you to be visible, it would be still nice, even without the database thing, it's still a lot of work.
> 
> I wanted to share with you this, even when I know, that this is not possible.

Make a feature request 😂
We used to save image urls in player's description but the problem was that that could be exploited, and it takes a lot of work to replace the images in all locations

**Comment by FrutyX** [comment]
>  the problem was that that could be exploited

Oh, good then we have a cybersecurity expert, right? I wouldn't mind hearing his opinion.

And I don't think we need to create another feature request, that will hang here for years, and never be addressed, if you wanted to do this, the info is right here.

**Comment by GramThanos** [comment]
Regarding the custom avatars, I liked a lot the initial idea by the old gladiatus tool addon, which allowed you to load different images for both guilds and players. It was really hard applying it everywhere, though the addon structure was not so advance back then. From my point of view the main problem is that it is a very nice way to recover the IP address of other players. This is also one of the reasons why I don't like the idea of auto loading images in messages and parsing links in private messages. In such cases you need a proxy server to hide the client's IP (just like google is doing on your emails or github on external images).

Regarding global arena, at some point if GameForge does not care to implement new features, why should I spend 10 hours coding for them when I can spend 100 and recreate a spin off game? I see more and more people requesting from our addon to develop features that GameForge should develop.

**Comment by FrutyX** [comment]
@GramThanos 
An interesting point of view, sadly, it is true.

There is a fine line between "for Gameforge" and "for the addon", it's a matter of perspective, I do see a conflict in this, GCA was made for Gladiatus, but you created the addon, you added all those features, you made it, are you coding for them? Like I've said, there is a conflict I cannot solve.

Gameforge does not listen to their players, so what is the next place they'll go? You know the answer, it's the only option left.

My whole goal was to use these avatars, but unfortunately, I don't see a easy way to use them.

Once we know Gladiatus is gonna truly end, you can prepare you spinoff thingy, I wanna see that :D.

**Comment by GreatApo** [comment]
The only solution to custom avatars is supporting only predefined pictures which are saved in the addon.

**Comment by GramThanos** [comment]
You can also allow trusted 3rd party sources (e.g. imgur) or image as a text (e.g. in base64 format). The later will be tricky in validating its sanity.

**Comment by Zitronenen** [comment]
I can't find the feature request for "Custom market prices" and i hope i am right here in the talk thread.
Is it possible to add an "On / Off" switch here?
It is possible that someone would like to have this switched on every now and then without having to rewrite their specific buttons

**Comment by FrutyX** [comment]
#166

**Comment by Zitronenen** [comment]
Hi there,

following feature "https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Market.md#toggle-button-changing-sell-price-always-to-1-gold"
I think the ON / OFF button label is incorrect.
"Activate special sales modes" would be better.

ON = 1G / Auto / Value and the new custom market prices

Ask for clarification or whether I should create a feature request

**Comment by SleepingShadow-bit** [comment]
Hey Guys,

https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Reports.md#load-each-reports-reward
We have here the same "problem" as with #279

**Comment by FrutyX** [comment]
@SleepingShadow-bit create a new issue for that with an detailed explanation.

**Comment by SleepingShadow-bit** [comment]
Done.

The new version is available on chrome. Thank you guys for your work =)

**Comment by SleepingShadow-bit** [comment]
https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Global.md#show-forgesmelt-timer-indicator

Do you also have the problem that the link does not lead to the desired post?
Did I do something wrong in the documentary?

**Comment by GreatApo** [comment]
> https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Global.md#show-forgesmelt-timer-indicator
> 
> Do you also have the problem that the link does not lead to the desired post? Did I do something wrong in the documentary?

The # in a link scrolls the page at the element with the target ID, however, when the images are loading the resulting scroll height is wrong... I would say this is a browser bug/issue.

**Comment by SleepingShadow-bit** [comment]
@GreatApo 
Should the documentation be adapted to the beta or only stable version?

Otherwise I will create a beta version that will be added afterwards

**Comment by GreatApo** [comment]
> @GreatApo Should the documentation be adapted to the beta or only stable version?
> 
> Otherwise I will create a beta version that will be added afterwards

I think the most efficient would be to follow the beta/development version to make it easier and simpler to maintain. Generally we add or improve features so it shouldn't be a huge problem if there are additional stuff in there.
Btw, don't spend time updating the existing images if there are small changes like the quality colors...

**Comment by SleepingShadow-bit** [comment]
My edit costs less time than your fabulous programming =)

**Comment by FrutyX** [comment]
And meanwhile @GramThanos:

_"They'll just do everything for me"_

![sitting-on-couch-al-bundy](https://user-images.githubusercontent.com/23278552/147890221-3fc2163d-6181-4125-8cd7-e7ed7a1cae13.gif)

🤣 🤣

**Comment by GramThanos** [comment]
So many emails... I can't even keep up...

**Comment by FrutyX** [comment]
Bruh, https://en.gladiatus-tools.com has been down since the new year (I think), now, during the costume event...

I hope he'll revive it, I remember when I was writing an email to Skars to revive his page, and he replied and revived the page :D. I don't know about Michalus tho.

**Comment by SleepingShadow-bit** [comment]
> Bruh, https://en.gladiatus-tools.com has been down since the new year (I think), now, during the costume event...
> 
> I hope he'll revive it, I remember when I was writing an email to Skars to revive his page, and he replied and revived the page :D. I don't know about Michalus tho.

On Discord from Michalus:
Planned to fix it today, but still in car. Fixing tools will have to wait till tomorrow. Sorry

**Comment by FrutyX** [comment]
Oh, great, thanks for letting me know.

**Comment by SleepingShadow-bit** [comment]
Do you play gladiatus with the gca addon on your mobile phone (browser Kiwi https://kiwibrowser.com/ )?

Because, some buttons do not work.
If the add-on is not programmed for the mobile version, please ignore my question

**Comment by FrutyX** [comment]
There are features broken on phones, GCA never mentioned anywhere that it does support phones.

There are too many variables on phones (OS version, browser and it's version etc. etc.). Supporting cross platform would be insanity, it's just my opinion, but you don't want to rewrite half of GCA, just because it doesn't work on phones. Unless you have a team of people experienced with mobile development. 

There is a reason, why extensions are not possible to use on Google mobile and you have to use browser like Yandex or Kiwi.

**Comment by SleepingShadow-bit** [comment]
> GCA never mentioned anywhere that it does support phones.

Never said otherwise =) just a question, i am noob about programming =) 
And thx for explaining =)

**Comment by GramThanos** [comment]
I personally use GCA on mobile. A while ago it was also available for Firefox mobile but after a big update Firefox removed extension support on mobile.

The addon should work without a problem on a mobile browser. Bugs observed on mobiles should only be related to either browser bugs or touch events bugs.
From my side, I noticed some touch events bugs on mobile, but they where gladiatus related.

**Comment by SleepingShadow-bit** [comment]
> From my side, I noticed some touch events bugs on mobile, but they where gladiatus related.

which browser do you use? 

**for example - bugs or anything else:**
- cant press the data and exp icon but the ugly gladiatus chat works 
- cant press the button at the smelt both of them
- its difficult to press the button add to target list 
- can't press the setting icon for Group inventory options

![IMG_20220104_013631](https://user-images.githubusercontent.com/8767665/147994850-6454c7df-4030-47c2-b500-e6fd9008466c.jpg)
![IMG_20220104_005132](https://user-images.githubusercontent.com/8767665/147994851-1b16faed-dca2-4cbe-9ab5-b98c6ccf3c6b.jpg)

**Comment by FrutyX** [comment]
I've encountered different bugs when I was testing this, and that's exactly what I've said - too many variables on phones.

But if Thanos and Apo wish to support it, then who am I to talk into this? They can fix it for phones, if they want to use their time on it, but just make sure you don't break it for desktops in the process.

**Comment by GreatApo** [comment]
I don play Gladiatus on mobile because I find it extremely difficult. Personally, I believe Gameforge should have either make a mobile version of the game or add a mobile css.

@SleepingShadow-bit it sounds like you do play on mobile and you are familiar with some of the problems. Some of the things you mentioned seem easy to fix with css, so I would suggest you open a new issue and copy paste your comments about the mobile usage there. Then we can have a look an what can be fixed.

**Comment by SleepingShadow-bit** [comment]
I am playing on PC and Mobilephone (but only with GCA Addon).
Alright, i open a bug issues

**Comment by GramThanos** [comment]
> Some of the things you mentioned seem easy to fix with css

I don't think so. In Kiwi, the buttons just don't work. A new `touchend` event is needed on every non interactive element (e.g. a `div`) that has an `click` event. I haven't found any other solution (well... you can also change the elements to `a`).

This is not only a problem of the addon, for example you can not even accept the cookie notice banner.

I am trying to find the source of the problem because technically this is a browser bug.

**Comment by FrutyX** [comment]
Hmm, Kiwi has a GitHub repo public, but there is this, even if you manage to fix this via a browser fix, this isn't the only mobile browser you can run for GCA, there is also Yandex and maybe more, I am not even aware of.

**Comment by FrutyX** [comment]
I am getting lost in all these "filter by" issues, so what I got from all off them, is that the advanced search filters in packages need a huge update (like a whole update dedicated only to it lol) and people want to filter by.... everything possible :D.

**Comment by FrutyX** [comment]
Hey boys, 

I don't know if you watch Gladiatus news, but there is a huge server merges planned, so the question is, is GCA ready for it? The extensions itself will just load it as a new profile settings I guess, however now when I am thinking about it, they've mentioned that you will be able to choose your own language, not sure how that will mess with GCA, because some translations are loaded directly from the game.

> - Players will be able to choose their own display and communication languages on the new servers.


The simulator and probably Global Arena will need some adjustments, because everything will change, well, hopefully it will not give you too much trouble...

If you still want to keep actively GCA in development, I very advise that you'll be here on an emergency duty next month, when they'll start rolling it, because who knows what is gonna happen to GCA, let's hope that nothing, but be prepared for anything.

Now for some GCA status update, it has been pretty much clear of any errors (except the ones that are already on GitHub), only Chrome is reporting the manifest version 3 in 2023, but that is already known.

We (well, you) might need to release a smaller hotfix version, if things go bad.

**Comment by GramThanos** [comment]
I am expecting many things to break down. I don't think language will be the biggest problem.

Personally, I quit Gladiatus a few months ago and I haven't decided if I will keep working on the addon. Gameforge doesn't deserve my time.

**Comment by FrutyX** [comment]
Yea, you've had similar opinions in the past, however you are not doing it for Gameforge, well, not only, the players are the ones that use it, not Gameforge.

Dunno what is Apo's opinion, I was thinking that I might stop with Gladiatus too after this, because I will be on the same server with so many communities that it will be plainly stupid, but I haven't 100% decided.

In the end, GCA was your free time project, not Gameforge's play toy.

**Comment by FrutyX** [comment]
You can make it a feature request, however Gameforge is not finished updating and fixing stuff, so we wait until their work is complete.

**Comment by SleepingShadow-bit** [comment]
I'm currently waiting for the bugs to be fixed.  Then please make a feature request.  Maybe via mouse over for example.
Now just wait

**Comment by FrutyX** [comment]
Boys, I can see you are both active, which is nice (hope you doing good), one important thing it would be nice to focus on is finishing the manifest v3, take a look at it when you have time, it's only a matter of time anyway, because in 2023 v3 is gonna be required.

I see there is a branch for it, if you need me for some testing, just tell me what exactly you want to know.

**Comment by GreatApo** [comment]
@GramThanos would you be able to have a look at the manifest at some point? I feel like this would be out of my waters...

**Comment by GramThanos** [comment]
I remember that I had prepared a manifest v3 and it was working. The reason I didn't swapped it was that manifest v3 works only on chrome. Thus, we will have to maintain a v2 for Firefox and others.

**Comment by FrutyX** [comment]
From Firefox blog:

> Work is continuing in parallel, and we expect to launch MV3 support for all users by the end of 2022.

I guess it's just a matter of time, seems like everyone is aiming for 2023, then good to be prepared.

**Comment by SleepingShadow-bit** [comment]
Do I understand correctly that the orange marked square are qucklinks?  
What is the thought of this?  An example Melefica linked to workbench

![Uploading IMG_20220701_001346.jpg…]()

**Comment by GreatApo** [comment]
> Do I understand correctly that the orange marked square are qucklinks? What is the thought of this? An example Melefica linked to workbench
> 

Remnants from when GF thought it is a good idea not to have the workbench and smelter as options on the menu

**Comment by SleepingShadow-bit** [comment]
Ooih really? GF :) 

Could it be cleaned up again (malefica and forge).  
Or is there another nicety for a quicklink? 

The auction house had a very nice link to mercenary supplies.

**Comment by warkadiusz** [comment]
Hi,

I hope that's the best place to ask this question.

Did you think about adding "+ package cost" button/switch in the guild market? I'm talking about packaging money for someone, I regularly forget about adding the cost, have to calculate it manually etc. I haven't found any past issues about it, but maybe there are GF-reasons not to implement it. I could happily do the implementation if that feature would be helpful for anyone.

**Comment by GramThanos** [comment]
Something like:
![image](https://user-images.githubusercontent.com/14858959/177188253-9203072a-4688-4c2f-9ba3-c395138a0e3f.png)
?

**Comment by GramThanos** [comment]
> I think that here only the fees are calculated without a pact. Correctly?
> 
> I like this feature and style.

I used GF's algorithm. I don't think it takes into consideration the packts. But I am not sure.

**Comment by warkadiusz** [comment]
> Something like:

yeah, exactly. That's even more than I was thinking of, as it also suggests some common packages sizes. For me calculating the cost would be already sufficient as the packages are quite random anyway. Is it already there and I missed it or you just made some quick concept here?

**Comment by SleepingShadow-bit** [comment]
@warkadiusz look at our documentation https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Market.md... "Custom market price"

@GramThanos pacts are included

**Comment by warkadiusz** [comment]
> @warkadiusz look at our documentation https://github.com/DinoDevs/GladiatusCrazyAddon/blob/master/documentation/features/Documentation%20Market.md... "Custom market price"

Thanks, this I definitely missed out.

> pacts are included
Yeah, if the cost is taken from what the game calculates then at least Edyl's pact is taken into account, discounts the cost by half

**Comment by GreatApo** [comment]
> Is it already there and I missed it or you just made some quick concept here?

Thanos just coded it. You would need to load the developer version if you want to test that now.

**Comment by warkadiusz** [comment]
Ok I see it's in master, pulling now and I'll test it. Thank you, that was quick!

**Comment by warkadiusz** [comment]
Might be related or not to this change, but as I've loaded the plugin some numbers in the game got concatenated with themselves, see here:
![dups](https://user-images.githubusercontent.com/13415737/177196784-08e5b2ea-a2ae-4aeb-a504-333e2c4c467d.png)

Works ok after refreshing page, just loading the plugin causes it.

**Comment by GreatApo** [comment]
> Might be related or not to this change, but as I've loaded the plugin some numbers in the game got concatenated with themselves, see here: ![dups](https://user-images.githubusercontent.com/13415737/177196784-08e5b2ea-a2ae-4aeb-a504-333e2c4c467d.png)
> 
> Works ok after refreshing page, just loading the plugin causes it.

You have 2 versions enabled. Disable the one from the store.

**Comment by warkadiusz** [comment]
> You have 2 versions enabled. Disable the one from the store.

It wasn't enabled, now I've completely removed it and this still happens. But anyway, that's just a detail because it happens only when loading the extension.

**Comment by GreatApo** [comment]
> > You have 2 versions enabled. Disable the one from the store.
> 
> It wasn't enabled, now I've completely removed it and this still happens. But anyway, that's just a detail because it happens only when loading the extension.

It is 100% because 2 GCA extensions are running (for some reason). It may be visually ok but it may cause other problems.

**Comment by FrutyX** [comment]
Hmmm, I don't see such problem with the latest commit.

**Comment by warkadiusz** [comment]
Anyway, the "add cost" feature works great, thank you once again.

**Comment by FrutyX** [comment]
Thanos, I am still waiting for your Gladiatus spin off, you gonna need a good graphics team tho 😘

**Comment by GramThanos** [comment]
![image](https://user-images.githubusercontent.com/14858959/177206206-38981177-06b5-48d9-8196-afc80c86da6c.png)
Weekend project on March ... but I got bored ...

**Comment by FrutyX** [comment]
loooooooooooooooooool

**Comment by GramThanos** [comment]
Here is a rat fight
![image](https://user-images.githubusercontent.com/14858959/177206386-3b5219d0-2bb5-4369-8f4d-c9de668cc449.png)

**Comment by FrutyX** [comment]
Gameforge is done for, boys 🤣

**Comment by FrutyX** [comment]
Everybody kneel and start praying to the GreatGoat!

![apo](https://user-images.githubusercontent.com/23278552/177765931-6382694c-6e6b-4178-b852-9426e49473aa.PNG)

**Comment by SleepingShadow-bit** [comment]
ok guys...i think i am a little bit ...alzheimer. Is the hover-effect at the message new? 
![Neues-Video](https://user-images.githubusercontent.com/8767665/178047228-49d50f58-e9ac-433b-8f11-f1604cf2493e.gif)

On the Kiwi Mobile Browser I have to long-click to activate the hover

**Comment by GreatApo** [comment]
> ok guys...i think i have a little bit ...alzheimer. Is the hover-effect at the message new? 
> 

That's old

**Comment by SleepingShadow-bit** [comment]
I don't know what was wrong about the mobile version.
Now all it's fine =)

**Comment by GramThanos** [comment]
If I remember correctly, when the addon detects that you are on mobile, the links on the messages are shown without the need to hover.

**Comment by FrutyX** [comment]
I do have a question, why is the item level showed in the auction list by a red indicator, when we can see the level on the item it self? Is that an old feature?

![auc](https://user-images.githubusercontent.com/23278552/179422451-1b60d88b-0c04-461f-8c47-e71bfca2604f.PNG)

**Comment by GramThanos** [comment]
Because GF stole the feature.

**Comment by FrutyX** [comment]
Ahahaah 🤣 , I guess you can put that on the extermination list. No need to load an feature that no longer serves a purpose.

**Comment by FrutyX** [comment]
4.3.1 is out, they've done something to images, GCA features does not load them, it's giving a 404 error, check Negotium or the add to target list + button for example, and what's what I only noticed with a quick look,.

**Comment by FrutyX** [comment]
Yeah, there are new URLs:

![img](https://user-images.githubusercontent.com/23278552/179516867-2d25ab47-3ade-4414-8ce9-55d556636bfd.PNG)

**Comment by FrutyX** [comment]
We have to wait for the merges anyway, because that will broke something too, Global Arena for sure.

**Comment by FrutyX** [comment]
Man, I have to say, Thanos is hella on drugs 😆

**Comment by FrutyX** [comment]
The first merges are coming tomorrow, wouldn't be good to disable Global Arena and the simulator before? Remotely? It will cause havoc in both, because old server will become invalid.

Or do you plan to keep up with it and live update it during the merges? I mean, that's only up to you how you manage it, I just know we don't need another thing that will break the extension.

**Comment by SleepingShadow-bit** [comment]
Those who are currently using the official version 4.3.5 currently have a destroyed addon.  I think wait, find and fix errors and release a 4.3.6 stable version as soon as possible

**Comment by FrutyX** [comment]
I said remotely Sleeping, Global Arena is running outside the extension.

**Comment by GreatApo** [comment]
I would leave them be for now (global arena and the simulator). Also, I will be busy this and next week (they pushed the merges so I couldn't plan ahead) so I don't know when the main fixes will be done.

**Comment by FrutyX** [comment]
We still have Thanos, don't we? Or he is a busy boy too? :D

**Comment by GreatApo** [comment]
Thanos is a tricky card. Either he will fix everything, or he will light the fire and let it burn XD

**Comment by FrutyX** [comment]
Oh, I have noticed that he is an... interesting guy :D.

**Comment by GreatApo** [comment]
> Oh, I have noticed that he is an... interesting guy :D.

He is just a bit fed up with Gladiatus and we both have more important stuff that we should be doing

**Comment by FrutyX** [comment]
Ah, that's fine and understandable, actually, sometimes I had a Thanos mood when it comes to Gladiatus, but that's just a mood, there has been done too much work and effort into GCA to just let it burn.

If only Gameforge was thinking the same way, but it's just a company that wants money in the end.

**Comment by GramThanos** [comment]
I told Apo to shut down the project (and the rest of them) months ago. I expressed my opinion multiple times.

I love the community we set up during the years, but we choose the wrong game and company to do so.

**Comment by FrutyX** [comment]
And also, when Apo seriously decides to burn it, I will not argue him, we'll lit the candle together.

**Comment by GreatApo** [comment]
Gameforge is doing what they believe is good for them, and if I was onboard I may have done the same. I am not going to argue on that over a 12y old game. But as Fruty said, we have done too much to just shut it down. Now if these merges need us to rewrite the whole addon... Fruty pass me the candle hahaha.

**Comment by FrutyX** [comment]
**--> 🕯️ <--**

**Comment by FrutyX** [comment]
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 

The first round of merges failed... how surprising, right. 

Now they will do some core code changes and we can literally get the candle, lol.

🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

**Comment by FrutyX** [comment]
Obviously I am just trolling Thanos, surely, SURELY nothing will be broken :).

**Comment by FrutyX** [comment]
![lol](https://user-images.githubusercontent.com/23278552/180053928-d9ec3708-8e5b-4cee-b9cb-40aa2f2e53c1.PNG)

And they want to merge servers... if the new one are not upgraded, it's gonna bloody die.

**Comment by FrutyX** [comment]
I know this was very far from being needed... but I brought to life the GitHub Wiki page.

Take a look Thanos, take a look at the fruits of your work.

**Comment by GramThanos** [comment]
I have to rewrite the troubleshooting:P

**Comment by SleepingShadow-bit** [comment]
Looks good. Thanks @FrutyX

**Comment by FrutyX** [comment]
> I have to rewrite the troubleshooting:P

![pepe](https://user-images.githubusercontent.com/23278552/180603609-5b5a1490-ba79-4a59-9710-da6cb5f69aab.png)

**Comment by GramThanos** [comment]
> > I have to rewrite the troubleshooting:P
> 
> ![pepe](https://user-images.githubusercontent.com/23278552/180603609-5b5a1490-ba79-4a59-9710-da6cb5f69aab.png)

Whaaat? There are some more tricks that you can do to fix problems.

**Comment by FrutyX** [comment]
If it's an enhancement, well, then sure.

**Comment by FrutyX** [comment]
Eh, boys? My nice plans got kinda wrecked.

I wanted to set up GitHub discussions, however when I try to set them up, it gives me an 404 error.

So my guess is, that I don't have the permissions for it, or they are disabled in the repo settings (you need to enable it).

I simply wanted and want to keep Issues issues, and Talk talk, GitHub has many interesting features and options, and since Apo some time ago stated me as the project Administrator, I still feel some responsibility for it, so I am improving it and I want to customize those discussions just simply for GCA.

**Comment by GramThanos** [comment]
I enabled them, let me know if you have any problem.

**Comment by FrutyX** [comment]
Thanks Thanos <3

**Comment by FrutyX** [comment]
> let me know if you have any problem.

I think I do, unfortunately, there must be something extra permission settings. I was playing with it in my own repo, and now for example, I can see I cannot even edit the categories, sorry to bother you :D, thought I'm gonna do all myself, unfortunately I am limited.

**Comment by FrutyX** [comment]
Or perhaps I won't be needing them (if there are any), so we can keep it simple and you do it.

Remove categories:

- Ideas (we keep feature requests in issues)
- Polls (not needed)
- Show and tell (not needed)

**Comment by GramThanos** [comment]
I promoted you 2 to maintainers. @FrutyX @SleepingShadow-bit 
I guess now you have more rights.

**Comment by FrutyX** [comment]
Ah, I now have what I needed, thank you @GramThanos ❤️ .

@SleepingShadow-bit careful 😆.

**Comment by FrutyX** [comment]
It's time to say this issue goodbye, from now on, please continue in #389 (GCA related talk).

#390 for general talk.

Closing.

**Comment by SleepingShadow-bit** [comment]
> @SleepingShadow-bit careful 😆.

Oooh shit xD
